modules/text_server_adv/text_server_adv.cpp

```diff
@@ -5698,6 +5697,7 @@ bool TextServerAdvanced::_shaped_text_update_breaks(const RID &p_shaped) {
 					}
 				}
 			}
+			ubrk_close(bi);
```

Comment:
It should only close it if open/clone was successful.

Reply:
I restored what it was previously:

https://github.com/godotengine/godot/blob/d2dfab1fef350eb30ff4901ca0c3351878cf6ced/modules/text_server_adv/text_server_adv.cpp#L5441-L5442

But yeah, I agree it didn't make sense then either, probably. I'll just put it into the `if` block.